src/Faker/Provider/et_ET/Person.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Faker\Provider\et_ET;
+
+class Person extends \Faker\Provider\Person
+{
+
+
+    protected static $titleMale = array('Härra', 'Hr.');
+
+    protected static $titleFemale = array('Proua', 'Pr.', 'neiu', 'preili');
+
+    /**
+     * @link http://www.eki.ee/books/ekk09/index.php?p=6&p1=5&id=548
```

Comment:
this site mentions a copyright. We need a source compatible with the MIT license.